packages/website/src/building-blocs/Demo.tsx

```diff
@@ -70,40 +70,77 @@ const Demo = ({children, snippet, center = false, grow = false, title}: DemoProp
                     {title}
                 </Title>
             ) : null}
-            <Group grow className={classes.sandbox} align="stretch" spacing={0}>
-                <Box component={center ? Center : 'div'} className={classes.preview}>
-                    <ScrollArea.Autosize maxHeight={MAX_HEIGHT}>
-                        <div className={classes.previewWrapper}>{children}</div>
-                    </ScrollArea.Autosize>
-                </Box>
-                <div className={classes.code}>
-                    <Prism
-                        withLineNumbers
-                        language="tsx"
-                        colorScheme="dark"
-                        getPrismTheme={(_theme, colorScheme) => (colorScheme === 'light' ? vsLight : vsDark)}
-                        radius={0}
-                        noCopy
-                        scrollAreaComponent={ScrollArea.Autosize}
-                        styles={{scrollArea: {maxHeight: MAX_HEIGHT, minHeight: MIN_HEIGHT}}}
-                    >
-                        {snippet}
-                    </Prism>
-                    <Stack className={classes.actions} spacing="xs">
-                        <Tooltip label={clipboard.copied ? 'Copied' : 'Copy'} position="left">
-                            <ActionIcon radius="sm" onClick={() => clipboard.copy(snippet)}>
-                                {clipboard.copied ? <CheckSize16Px height={16} /> : <CopySize16Px height={16} />}
-                            </ActionIcon>
-                        </Tooltip>
+            {layout === 'vertical' ? (
```

Comment:
That's a lot of duplicated code which in the long run will cause issue

I suggest you try out https://mantine.dev/core/flex/ instead of the group/stack

Reply:
nice thats much better thanks for Flex :P